.github/workflows/psalm.yml

```
@@ -39,7 +39,7 @@ jobs:
         run: |
           echo "::group::modify composer.json"
           composer remove --no-update --no-interaction symfony/phpunit-bridge
-          composer require --no-update psalm/phar phpunit/phpunit php-http/discovery psr/event-dispatcher
+          composer require --no-update psalm/phar phpunit/phpunit:@stable php-http/discovery psr/event-dispatcher
```

```suggestion
          composer require --no-update psalm/phar phpunit/phpunit:^9.5 php-http/discovery psr/event-dispatcher
```

I'm not sure we will be compatible with PHPUnit 10 right away. Let's pick a version we actually run our tests with.

---

Good idea.